starboard/android/shared/player_components_factory.h

```diff
@@ -423,7 +447,11 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
       SB_DCHECK(audio_renderer_sink);
 
       using starboard::shared::starboard::media::AudioStreamInfo;
-      auto decoder_creator = [enable_flush_during_seek](
+      bool audio_use_mediacodec_callback_thread =
```

Comment:
As dropping frames is likely to happen on video decoder, instead of audio decoder, do you think we can only move video mediacodec callback with a handler thread?

If yes, would you be able to verify if we only move video mediacodec callback with a handler thread can solve the issue?